src/link/inline.js

```diff
@@ -4,7 +4,7 @@ import {
 	Button,
 	KeyboardShortcuts,
 } from '@wordpress/components';
-import { getTextContent, slice, useAnchor } from '@wordpress/rich-text';
+import { getTextContent, slice, useAnchorRef } from '@wordpress/rich-text';
```

Comment:
I replaced anchorRef with anchor everywhere in a previous PR because anchorRef is deprecated.

Does it appear to make a real difference here?

Reply:
Thanks for the comment @stephanebisson u remind me of this issue, this is exactly the problem where we got this jumping bug, as anchor accepts Element|Virtual Elements

with the current solution, we use `useAnchor` `placement` correctly with the latest API

Thanks a lot